eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -6,6 +6,21 @@ jobs:
   condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
   strategy:
     matrix:
+      Linux_ARM:
+        OSVmImage: 'ubuntu-18.04'
+        vcpkg.deps: ''
+        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
+        AptDependencies: 'gcc-arm-none-eabi'
+        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake'
+        AZ_SDK_C_NO_SAMPLES: 'true'
+      Linux_ARM_no_preconditions:
+        OSVmImage: 'ubuntu-18.04'
+        vcpkg.deps: ''
+        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
```

Comment:
ditto, arm64-linux

Reply:
I've removed the triplet because it's not necessary (we aren't using vcpkg)